content/sensu-core/0.29/guides/troubleshooting.md

```diff
@@ -304,6 +304,9 @@ In the output above, we're specifically interested in the `TLS Web Server Authen
 
 ![ssl_example][11]
 
+#### Unknown CA
+There is a possibility that you may encounter an error inside of RabbitMQ when configuring SSL/TLS that states the following: "Unknown CA". To remedy this issue, ensure that the _full certificate chain_ is present on every client connecting to RabbitMQ (e.g., Sensu clients, Sensu Servers.)
```

Comment:
Is there another word besides "client" we can use here? It seems odd to describe the Sensu Server as a client.

Reply:
Ehhhh, not really? They're clients to the RabbitMQ instance. I don't know if @cwjohnston might have a better suggestion, but since from a technical standpoint they are clients, I'm having a hard time thinking of another term :/